lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml

```diff
@@ -350,8 +388,10 @@ PYI_NEW_DETAILS_PAGE:
       targetState:
         PYI_CONFIRM_DELETE_DETAILS_PAGE
     end:
-      targetState:
-        IPV_IDENTITY_REUSE_PAGE
+      targetState: IPV_IDENTITY_REUSE_PAGE
+      checkFeatureFlag:
+        ticfCriBeta:
+          targetState: CRI_TICF_ON_REUSE
```

Comment:
I thnk if we get to this state, we will have already called the TICF CRI on a reuse journey. Adding this here would result in calling it twice. Maybe that's ok though, and would prevent us accidentally missing it iof the config changes in future.

Reply:
Calling ticf twice is ok alrady been confirm by David (and informed TICF).

Reply:
Calling ticf twice is ok alrady been confirm by David (and been informed to ticf team).